core/src/main/java/hudson/lifecycle/WindowsInstallerLink.java

```diff
@@ -310,6 +311,8 @@ static int runElevated(File jenkinsExe, String command, TaskListener out, File p
         } finally {
             try (InputStream fin = Files.newInputStream(new File(pwd,"redirect.log").toPath())) {
                 IOUtils.copy(fin, out.getLogger());
+            } catch (InvalidPathException e) {
+                // ignore;
```

Comment:
AIUI this is code running on the remote JVM so sending to the logger is probably akin to `/dev/null`